docs/entity_registry_index.md

```diff
@@ -3,7 +3,7 @@ title: Entity Registry
 sidebar_label: Introduction
 ---
 
-The entity registry is a registry where Home Assistant keeps track of entities. Any entity that is added to Home Assistant and has a unique ID will be registered in the registry.
+The entity registry is a registry where Home Assistant keeps track of entities. Any entity that is added to Home Assistant which specifies the [`unique_id` attribute](https://developers.home-assistant.io/docs/core/entity#generic-properties) will be registered in the registry.
```

Comment:
We should probably use an internal link rather than external to be consistent with other pages
```suggestion
The entity registry is a registry where Home Assistant keeps track of entities. Any entity that is added to Home Assistant which specifies the [`unique_id` attribute](/core/entity.md#generic-properties) will be registered in the registry.
```